charts/agent/README.md

```diff
@@ -166,6 +166,14 @@ Chart to install K8s collection stack based on Observe Agent
 | cluster.events.enabled | bool | `true` |  |
 | cluster.events.pullInterval | string | `"20m"` |  |
 | cluster.metrics.enabled | bool | `true` |  |
+| cluster.metrics.pod.action | string | `"keep"` |  |
```

Comment:
I'm actually not sure this even belongs nested under cluster at all. might make sense to break this out to a new top level section called `application`

Reply:
put it there because thats where the other switches for that container are - no quarrels with moving it out - we'll have to make sure we deal with enabled switch in chart but that should be np